meinberlin/apps/votes/models.py

```diff
@@ -67,6 +71,48 @@ def project(self):
     def is_valid_for_item(self, item):
         return item.module == self.module
 
+    @staticmethod
+    def get_hash_for_token(token, module):
+        params = argon2.profiles.RFC_9106_LOW_MEMORY
+        salt = module.slug
```

Comment:
this uses the module slug as salt for a little bit of extra security, however this assumes that the slug is never changed. Is that ok? otherwise we would need a hashing algorithm without salt like sha512 (which is less suitable for storing confidential things)